### ISSUE TITLE
not.finished file created even though download completed

### ISSUE THREAD

**Issue by ilovezfs** [Evolution0/bandcamp-dl#119]
**Python version:**  3.6.1

**Bandcamp-dl version:** 0.0.8-03

```
bandcamp-dl --artist=iamsleepless --album=rivulets
```
and
```
bandcamp-dl https://iamsleepless.bandcamp.com/track/under-the-glass-dome
```
are leaving behind a `0.0.8-03.not.finished` file even though the download completes.

This is causing the Homebrew test block to fail because it runs both commands, and so the second command attempts to resume the download of the first file, which already exists.

```
iMac-TMP:homebrew-core joe$ brew test -vd bandcamp-dl
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FormulaLoader): loading /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/bandcamp-dl.rb
Testing bandcamp-dl
==> Using the sandbox
/usr/bin/sandbox-exec -f /tmp/homebrew20170521-32870-dnti7v.sb /usr/local/Homebrew/Library/Homebrew/vendor/portable-ruby/2.0.0-p648/bin/ruby -W0 -I /usr/local/Homebrew/Library/Homebrew -- /usr/local/Homebrew/Library/Homebrew/test.rb /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/bandcamp-dl.rb -vd
/usr/local/Homebrew/Library/Homebrew/test.rb (Formulary::FromPathLoader): loading /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/bandcamp-dl.rb
==> /usr/local/Cellar/bandcamp-dl/0.0.8-03/bin/bandcamp-dl --artist=iamsleepless --album=rivulets
(1/1) [==================================================] :: Finished: 01 - rivuletsets
==> /usr/local/Cellar/bandcamp-dl/0.0.8-03/bin/bandcamp-dl https://iamsleepless.bandcamp.com/track/under-the-glass-dome
File: 01 - rivulets.mp3 already exists and is complete, skipping..
/usr/local/Homebrew/Library/Homebrew/debrew.rb:11:in `raise'
MiniTest::Assertion: Failed assertion, no message given.
1. raise
2. ignore
3. backtrace
4. irb
5. shell
Choose an action: 5
When you exit this shell, you will return to the menu.
bash-3.2$ cat 0.0.8-03.not.finished 
{'--album': 'rivulets', '--artist': 'iamsleepless', '--base-dir': None, '--embed-art': False, '--embed-lyrics': False, '--full-album': False, '--group': False, '--help': False, '--no-art': False, '--no-slugify': False, '--overwrite': False, '--template': '%{artist}/%{album}/%{track} - %{title}', '--track': None, '--version': False, '<url>': None, 'url': False}bash-3.2$ 
bash-3.2$
```

**Comment by Evolution0** [comment]
I know exactly what is wrong, me being an idiot.

Forgot to update the portion of the code that deletes the not.finished file when the download succeeds to account for the new filename based on version. Should be fixed in a few minutes.

Edit: Done, everything should work correctly now.

**Comment by ilovezfs** [comment]
@Evolution0 Thanks! I've shipped 0.0.8-04 in Homebrew now :)
